applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -125,6 +132,32 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	/* Since the audio datapath thread is preemptive, no actions on the
 	 * FIFO can happen whilst in this handler.
 	 */
+	bool data_size_mismatch = false;
+	static struct rx_stats rx_stats[AUDIO_CH_NUM];
+
+	rx_stats[channel_index].recv_cnt++;
+	if (data_size != desired_data_size && bad_frame != true) {
+		data_size_mismatch = true;
+		rx_stats[channel_index].data_size_mismatch_cnt++;
+	}
+
+	if (bad_frame) {
+		rx_stats[channel_index].bad_frame_cnt++;
+	}
+
+	for (int i = 0; i < AUDIO_CH_NUM; i++) {
```

Comment:
Do we want this for loop, or do we only want to print for the current channel_index?

Reply:
I added both, as this was the way it was done earlier. Makes it easier to see that the inactive channel is operating correctly.

Reply:
I do not quite understand the answer. Now the for loop is removed, so I guess we dont want it?